package.json

```diff
@@ -32,18 +32,18 @@
   },
   "homepage": "https://github.com/krakenjs/bundalo",
   "devDependencies": {
+    "dustjs-linkedin": "~2.7.2",
```

Comment:
Unfortunately the dust module has been known to release breaking changes in minor releases.